mqtt_transport.go

```diff
@@ -85,6 +85,14 @@ func (mh *MqttTransport) SetCertDir(certDir string) {
 	mh.certDir = certDir
 }
 
+func (mh *MqttTransport) Options() *MQTT.ClientOptions {
```

Comment:
do we want to expose MQTT.ClientOptions? it seems like an implementation detail that we would want to encapsulate

Reply:
Would be nice, but I guess it requires more changes...
```
mh.client = MQTT.NewClient(mh.mqttOptions) is called in constructor, so we need to call this lince explicitly again to change options (handlers and so on...) I propsed this change as it is pretty urgent, I am open to make it in another way so as not to expose Options but it will trigger more changes and some refactoring. Or maybe there is anotherway @boguslaw-wojcik  ?